modules/pel/peltool/peltool.py

```diff
@@ -386,6 +393,11 @@ def main():
     if args.non_serviceable:
         config.non_serviceable_only = True
 
+    if args.critSysTerm:
+        config.critSysTerm = True
+        global critSysTermSeverity
```

Comment:
I feel we could use https://github.com/ibm-openbmc/openpower-pel-parsers/blob/peltool-testing/modules/pel/peltool/pel_types.py to define some enum for this purpose since you are going to add a option to get PELs based on the severity.

Reply:
Yea, I think that makes sense.  Otherwise, looks good!